### PR TITLE
Add synchronizationService

### PR DESCRIPTION
**Pull request by rjzondervan**

### CODE REVIEW

File: Installation/Action/synczaken.action.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "title": "ZgwToVrijbrpBirthAction",
+  "$id": "https://vrijbrp.nl/action/vrijbrp.synczaken.action.json",
+  "$schema": "https://json-schema.org/draft/2020-12/action",
+  "version": "0.0.2",
```
Comment (WilcoLouwerse) [inline Installation/Action/synczaken.action.json:5]: 0.0.2 for a new action?

File: Installation/Action/synczaken.action.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "title": "ZgwToVrijbrpBirthAction",
+  "$id": "https://vrijbrp.nl/action/vrijbrp.synczaken.action.json",
```
Comment (WilcoLouwerse) [inline Installation/Action/synczaken.action.json:3]: different domain than mappings from https://github.com/CommonGateway/VrijBRPToZGWBundle/pull/1/files
might be better to be consistent with these references

File: src/ActionHandler/SynchronizeCollectionHandler.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+use App\ActionHandler\ActionHandlerInterface;
+use CommonGateway\VrijBRPToZGWBundle\Service\NewSynchronizationService;
+use CommonGateway\VrijBRPToZGWBundle\Service\VrijBrpService;
+
+class SynchronizeCollectionHandler implements ActionHandlerInterface
+{
+
+
+    /**
+     * @param NewSynchronizationService $synchronizationService
```
Comment (WilcoLouwerse) [inline src/ActionHandler/SynchronizeCollectionHandler.php:14]: Doc comment for parameter "$vrijBrpService" missing

File: src/ActionHandler/SynchronizeCollectionHandler.php [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+use App\ActionHandler\ActionHandlerInterface;
+use CommonGateway\VrijBRPToZGWBundle\Service\NewSynchronizationService;
+use CommonGateway\VrijBRPToZGWBundle\Service\VrijBrpService;
+
+class SynchronizeCollectionHandler implements ActionHandlerInterface
+{
+
+
+    /**
```
Comment (WilcoLouwerse) [inline src/ActionHandler/SynchronizeCollectionHandler.php:13]: Missing short description in doc comment

File: src/ActionHandler/SynchronizeCollectionHandler.php [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+     * @param NewSynchronizationService $synchronizationService
+     */
+    public function __construct(
+        private readonly NewSynchronizationService $synchronizationService,
```
Comment (WilcoLouwerse) [inline src/ActionHandler/SynchronizeCollectionHandler.php:17]: Avoid excessively long variable names like $synchronizationService. Keep variable name length under 20.

File: src/ActionHandler/SynchronizeCollectionHandler.php (hunk 3 above)
Comment (WilcoLouwerse) [inline src/ActionHandler/SynchronizeCollectionHandler.php:14]: Missing parameter comment

File: src/ActionHandler/SynchronizeCollectionHandler.php [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    ) {
+
+    }//end __construct()
+
+
+    /**
+     *  This function returns the requered configuration as a [json-schema](https://json-schema.org/) array.
+     *
+     * @throws array a [json-schema](https://json-schema.org/) that this  action should comply to
+     */
```
Comment (WilcoLouwerse) [inline src/ActionHandler/SynchronizeCollectionHandler.php:28]: Missing @return tag in function comment

File: src/ActionHandler/SynchronizeCollectionHandler.php [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    public function getConfiguration(): array
+    {
+        return [
+            '$id'         => 'https://commongateway.nl/ActionHandler/SynchronizationCollectionHandler.ActionHandler.json',
+            '$schema'     => 'https://docs.commongateway.nl/schemas/ActionHandler.schema.json',
+            'title'       => 'SynchronizationCollectionHandler',
+            'description' => '',
+            'required'    => [],
+            'properties'  => [],
```
Comment (WilcoLouwerse) [inline src/ActionHandler/SynchronizeCollectionHandler.php:37]: Missing properties in Handler here

File: src/Service/VrijBrpService.php [hunk 8]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace CommonGateway\VrijBRPToZGWBundle\Service;
+
+class VrijBrpService
```
Comment (WilcoLouwerse) [inline src/Service/VrijBrpService.php:5]: Missing class doc comment

File: src/Service/VrijBrpService.php [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+{
+
+
+    public function setVrijBRPDefaults(array $configuration): array
```
Comment (WilcoLouwerse) [inline src/Service/VrijBrpService.php:9]: Missing doc comment for function setVrijBRPDefaults()

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: Installation/Action/synczaken.action.json (hunk 1 above)
Comment (rjzondervan) [reply Installation/Action/synczaken.action.json:5]: Will be fixed in PR #3 or #4

File: src/ActionHandler/SynchronizeCollectionHandler.php (hunk 3 above)
Comment (rjzondervan) [reply src/ActionHandler/SynchronizeCollectionHandler.php:14]: Fixed in PR #4

**Review by bbrands02 — APPROVED** [review]

File: src/ActionHandler/SynchronizeCollectionHandler.php (hunk 4 above)
Comment (rjzondervan) [reply src/ActionHandler/SynchronizeCollectionHandler.php:13]: Fixed in PR #4

File: src/ActionHandler/SynchronizeCollectionHandler.php (hunk 5 above)
Comment (rjzondervan) [reply src/ActionHandler/SynchronizeCollectionHandler.php:17]: Fixed in PR #4

File: src/ActionHandler/SynchronizeCollectionHandler.php (hunk 3 above)
Comment (rjzondervan) [reply src/ActionHandler/SynchronizeCollectionHandler.php:14]: Fixed in PR #4

File: src/Service/VrijBrpService.php (hunk 8 above)
Comment (rjzondervan) [reply src/Service/VrijBrpService.php:5]: Fixed in PR 4

File: src/Service/VrijBrpService.php (hunk 9 above)
Comment (rjzondervan) [reply src/Service/VrijBrpService.php:9]: Fixed in PR 4

File: src/ActionHandler/SynchronizeCollectionHandler.php (hunk 6 above)
Comment (rjzondervan) [reply src/ActionHandler/SynchronizeCollectionHandler.php:28]: Added in PR #4

File: src/ActionHandler/SynchronizeCollectionHandler.php (hunk 7 above)
Comment (rjzondervan) [reply src/ActionHandler/SynchronizeCollectionHandler.php:37]: Will fix this in PR 4

**Pull request closed without merging**